modules/ROOT/pages/_partials/mule-runtime-metrics.adoc

```diff
@@ -46,41 +46,62 @@ The maximum number of concurrent flows across all Mule apps and business groups
 // end::cards[]
 
 // tag::table[]
-Business Group:: 
-The business group in which a Mule app is deployed.
 
-Environment:: 
-The environment in which a Mule app is deployed.
+Select the *Business Group Details* tab to view:
 
-Application::
-The name of the Mule application.
+[%header%autowidth.spread]
+|===
+|Field | Description
+
+|Business Group |The business group a Mule app is deployed to
```

Comment:
Suggest removing the "The" from these descriptions - for easier scanning.

Also, if they're all fragments, remove the periods.